doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -159,9 +159,11 @@ and ``coqtop``, unless stated otherwise:
   short or partially qualified names.
 
   .. seealso:: Section :ref:`names-of-libraries`.
-:-top *dirpath*: Set the toplevel module name to :n:`@dirpath` instead of ``Top``.
-  Not valid for `coqc` as the toplevel module name is inferred from the
-  name of the output file.
+:-top *dirpath*: Set the module name to :n:`@dirpath` for a `coqtop`
+  interactive session. If no module name is specified, `coqtop` will
+  default to ``Top``. This option is not accepted by `coqc` as the
+  toplevel module name is inferred there from the name of the input
+  file.
```

Comment:
```suggestion
:-top *dirpath*: Set the toplevel module name to :n:`@dirpath` for a `coqtop`
  interactive session. If not specified, the default is ``Top``.
  This option is not accepted by `coqc` because
  it infers the module name from the name of the input
  file.
```

Is "toplevel" essential here?

Reply:
indeed, tweaked the whole statement